taskcluster/ci/toolchain/kind.yml

```diff
@@ -1,11 +1,46 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
 ---
 loader: taskgraph.loader.transform:loader
+
 transforms:
-    - taskgraph.transforms.index_search:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.cached_tasks:transforms
     - taskgraph.transforms.task:transforms
 
+kind-dependencies:
+    - fetch
+
+
 jobs:
+    linux64-android-sdk-linux-repack:
```

Comment:
We might want to move these jobs definitions in separate files, since they're really different one another. I see that's what mozilla-central does anyway.

Reply:
This seems like a good idea. Let's do that here.

Reply:
Okay, done!